overview/encryption.md

```diff
@@ -96,75 +93,65 @@ yields
+- Adding `projectId` to the `componentId` changes the prefix to `KBC::ProjectSecure::`, making the cipher decryptable within
+the project's component configurations. This is recommended for **all secrets** used within a typical Keboola Connection project.
 
-- If both the component id and project id are provided, then the cipher starts with `KBC::ProjectSecure::` and it
-can be decrypted in all configurations of the given component within the given project. This is **recommended for all secrets** 
-used within a typical Keboola Connection project.
+- Providing all three IDs (`componentId`, `projectId`, `configId`) generates a cipher starting with
+`KBC::ConfigSecure::`, limiting decryption to a specific configuration. This is useful for preventing the copying of configurations.
 
-- If branch type is added to the both the component id and project id, then the cipher starts with `KBC::BranchTypeSecure::` and it
-can be decrypted in all configurations of the given component within the given project either in the default production branch or in any of 
-the development branches. This means that such encrypted value is not transferrable from production to development (and vice versa).
-Notice that it is not possible to encrypt value for only a single development branch.
+- Using only `projectId` yields a cipher that begins with `KBC::ProjectWideSecure::`, decryptable across the project's configurations.
+This cipher type helps encrypt information shared across multiple components, e.g., SSH tunnel settings.
 
-- If all three IDs (component id, project id and configuration id) are provided, then the cipher starts with
-`KBC::ConfigSecure::` and it can be decrypted only within the given configuration of the given component in the given project.
-This type of cipher is useful when you want to prevent copying a configuration.
+- Adding `branchType` restricts the encryption to the default production branch or to development branches. This means an encrypted value with this setting cannot be moved between production and development branches or vice versa. It is not possible to encrypt a value for just one development branch.
 
-- If branch type is added to all three IDs (component id, project id and configuration id), then the cipher starts with `KBC::BranchTypeConfigSecure::` and it
-can be decrypted and it can be decrypted only within the given configuration of the given component in the given project either in the default production 
-branch or in any of the development branches. This means that such encrypted value is not transferrable from production to development (and vice versa).
-Notice that it is not possible to encrypt value for only a single development branch.
+	- Using `branchType` with `componentId` and `projectId` results in a cipher beginning with `KBC::BranchTypeSecure::`. This allows decryption in any production or development configuration of the specified component in the project .
```

Comment:
Tady to to je taky trošku misleading - "decryption in any production or development configuration" vypadá jako že to jde dekryptovat kdekoliv, ale ono to jde dekryptovat buď jen v produkční nebo jen ve všech dev konfiguracích (podle toho co bylo zadaný v branchType)